tests/login.js

```diff
@@ -77,4 +77,28 @@ test(`Login with invalid username and password`, async t => {
 
     await t.expect(receivedError).eql(expectedLoginError)
     await t.expect(isErrorVisible).eql(true)
-});
\ No newline at end of file
+});
+
+import { Selector, t, ClientFunction }  from "testcafe";
```

Comment:
imports go to the top of the page, but these are already imported, so it's not necessary. VSCode should have alerted a double declaration here.